exercises/practice/grains/.docs/instructions.md

```diff
@@ -7,7 +7,7 @@ The king promised to pay whatever the servant could dream up.
 Knowing that the king loved chess, the servant told the king he would like to have grains of wheat.
 One grain on the first square of a chess board, with the number of grains doubling on each successive square.
 
-There are 64 squares on a chessboard (where square 1 has one grain, square 2 has two grains, and so on).
+There are 64 _(But because of language limitations, we take 31 in batch testing)_ squares on a chessboard (where square 1 has one grain, square 2 has two grains, and so on).
```

Comment:
We shouldn't update the upstream docs directly. The instructions append is the correct way to add track-specific information.

Reply:
Removed in new commit.